### PR TITLE
(Bugfix  #115) Loading spinner position not centered

### LINKED ISSUE #115: Loading spinner doesn’t show centered in the view
Instead of showing centered in the content view, the loading spinner is half obscured by the header bar. This is likely because the container of the spinner is not very tall → should stretch to viewport height.

### PR DESCRIPTION
**Pull request by Mikescops**
Fix #115. Add more spacing on top of the spinner.

### CODE REVIEW

File: src/Photos.vue [hunk 1]
```diff
@@ -82,6 +82,10 @@ export default {
 	flex-grow: 1;
 	flex-direction: column;
 	align-content: space-between;
+
+	&.icon-loading::after {
+		margin-top: 16px;
```
Comment (skjnldsv) [inline src/Photos.vue]: It's probably safer to make sure the container have a min height of 100%? :)

Comment (Mikescops) [reply src/Photos.vue]: What you mean ?

Comment (skjnldsv) [reply src/Photos.vue]: That the icon loading is centered on the parent space.
The real issue is that the parent (the app container) should have a minimum height of 100%

Comment (Mikescops) [reply src/Photos.vue]: Oh got it!

**Pull request merged**